### PR TITLE
Adds frameguard to add x-frame-options for any iframe requests

### PR DESCRIPTION
**Pull request by ajainarayanan**
Build: https://builds.cask.co/browse/CDAP-UDUT387

### CODE REVIEW

File: cdap-ui/server/express.js [hunk 1]
```diff
@@ -203,6 +204,7 @@ function makeApp(authAddress, cdapConfig, uiSettings) {
       maxAge: 60 * 60 * 24 * 365, // one year in seconds
     })
   );
+  app.use(frameguard({ aciton: 'sameorigin' }));
```
Comment (itsanudeep) [inline cdap-ui/server/express.js]: Typo? action, right?

Comment (ajainarayanan) [reply cdap-ui/server/express.js]: 🤦‍♂ I can do better. Fixed.

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by ajainarayanan** [comment]
Green build: https://builds.cask.co/browse/CDAP-UDUT387-6

**Pull request merged**